deploy/crds/enterprise.splunk.com_indexerclusters_crd.yaml

```diff
@@ -2309,13 +2364,37 @@ spec:
               description: Indicates whether the master is ready to begin servicing,
                 based on whether it is initialized.
               type: boolean
+            smartstore:
+              description: Splunk Smart store configuration
+              properties:
+                indexes:
+                  items:
+                    description: IndexSpec defines Splunk indexes configuration
+                    properties:
+                      indexName:
+                        type: string
+                      location:
+                        type: string
+                    type: object
+                  type: array
+                volumes:
+                  description: List to accommodate multiple volumes. For now, a hard
+                    check is added such that only one volume is supported to accommodate
+                    one common secrete object
+                  items:
+                    description: S3VolumeSpec defines the remote storage volume configuration
+                    properties:
+                      endpoint:
+                        type: string
+                      volumeName:
+                        type: string
+                    type: object
+                  type: array
+              type: object
           type: object
       type: object
   version: v1alpha3
   versions:
   - name: v1alpha3
     served: true
     storage: true
-  - name: v1alpha2
```

Comment:
Why remove v1alpha2? I think this prevents someone from upgrading v2 to v3.

Reply:
Thanks Mike. Yesterday, we were discussing this.  Actually, I didn't do this change, and it was auto-generated when i generated the CRDs with the smartstore config. I will revert it.

Reply:
Understood. Thought I added something to build/package.sh so that it would add this back into each of the CRD yamls, but looks like I did not. Would you be able to look into that? I think it'll just take a few extra "yq" commands, and then we don't have to worry about it getting clobbered again (we'll need to update it for each version bump, though).

Reply:
Sure Mike, i will take a look at it